tests/contest/contest/src/tests/lifecycle/container_lifecycle.rs

```diff
@@ -86,6 +97,31 @@ impl ContainerLifecycle {
             &self.container_id,
         )
     }
+
+    pub fn waiting_for_status(
```

Comment:
This function is duplicated with https://github.com/youki-dev/youki/pull/3082/files#diff-7c81e28b8064c3c2c13f68c038ca10661530915ffd2159b939e1dba7684b40e8

Reply:
The PR has been merged.